src/app/models/evt-models.ts

```diff
@@ -331,3 +331,60 @@ export class Word extends GenericElement {
 export class Deletion extends GenericElement {
     rend: string;
 }
+
+export class MsDesc extends GenericElement {
+    id? : string;
```

Comment:
Why is the id optional?

Reply:
I would also set it as mandatory and create one (maybe from the XML path) if none exists in data source

Reply:
There is already the function getID(). I have seen that it is already used